projects/packages/search/src/dashboard/components/pages/sections/plan-usage-section.jsx

```diff
@@ -10,14 +11,17 @@ const PlanUsageSection = props => {
 	if ( ! props.isVisible ) {
 		return null;
 	}
+	// TODO: Add logic for plan limits.
+	const upgradeMessage = undefined;
 	return (
 		<div className="jp-search-dashboard-wrap jp-search-dashboard-meter-wrap">
 			<div className="jp-search-dashboard-row">
 				<div className="lg-col-span-2 md-col-span-1 sm-col-span-0"></div>
 				<div className="jp-search-dashboard-meter-wrap__content lg-col-span-8 md-col-span-6 sm-col-span-4">
 					<PlanSummary />
 					<UsageMeters />
-					<UsageMetersAbout />
+					<CUTWrapper type={ upgradeMessage } />
```

Comment:
Two things here:
1. I'm slightly against proliferating more acronyms in code, which makes the overall codebase much less approachable to fresh eyes.
2. `CUTWrapper` should be `cutWrapper` according to [our coding standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#abbreviations-and-acronyms) (see "Acronyms [...] at the start of a variable name"), but that honestly makes the component even harder to understand.

Since this is just a nitty, syntactic suggestion, let's address this in a follow-up PR.

Reply:
In this case, wouldn't we consider CUTWrapper to be more like a Class than a variable? Our docs note examples of `domDocument` for a var and `DOMDocument` for a class. My first thought is that a component is more akin to a Class conceptually.

Either way, I'll update the code to avoid proliferating more acronyms and rename the component. I agree that something with a less jargony name is probably better for code readability.

Reply:
Ah, good catch. Yes, you're right —  CUTWrapper is a class instead of a variable. My second point about coding standards is moot, in this case 🙂

Reply:
Ah, good catch. Yes, you're right —  CUTWrapper is a class instead of a variable. My second point about coding standards is moot, in this case 🙂